tests/unit_tests/data_steward/validation/main_test.py

```diff
@@ -412,30 +408,26 @@ def test_process_hpo_ignore_dirs(
         main.process_hpo('noob', force_run=True)
```

Comment:
'noob' is used 7 times here and it can be difficult to follow exactly what 'noob' is.  I would suggest prefixing the attribute/object with "fake" or "mock" (mock_id, fake_csv.csv, etc.)